actions/team_actions.jsx

```diff
@@ -119,13 +124,19 @@ export function addUserToTeamFromInvite(data, hash, inviteId, success, error) {
 export function addUsersToTeam(teamId, userIds, success, error) {
     TeamActions.addUsersToTeam(teamId, userIds)(dispatch, getState).then(
         (teamMembers) => {
-            if (teamMembers && success) {
-                success(teamMembers);
+            if (teamMembers) {
+                getChannelStats(ChannelStore.getCurrentId())(dispatch, getState).then(
+                    () => {
+                        if (success) {
+                            success();
+                        }
+                    }
+                );
```

Comment:
Sorry, one more thing. Can we move the success check and call out of the .then()'s? The success callback is for when the users were added successfully, the getting of stats can be done asynchronously and shouldn't have an effect on whether the success() callback is hit or not

Reply:
done, hope did that right